RockLib.HealthChecks.HttpModule/HealthCheckHttpModule.cs

```diff
@@ -53,19 +53,26 @@ public void Init(HttpApplication context)
     {
         _healthCheckRouteRegex = new Regex($@"^\/*({Route})\/*$", RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
+#pragma warning disable CA1031 // Do not catch general exception types
         try
         {
+            if(context == null)
```

Comment:
I'm also starting to do this:

```csharp
#if NET_60_OR_GREATER
ArgumentNullException.ThrowIfNull(context);
#else
if (context is null) { throw new ArgumentNullException(nameof(context));
#endif
```

The syntax may be off, but hopefully this conveys the idea.

Reply:
Your null check should be outside the `try/catch` block, otherwise it'll just get swallowed up.

Reply:
is `is null` just shorter than using `== null` or does it do something different?

Reply:
It is a literal check for `null`. `== null` uses the equality operator, and that can be changed. Most developer will never add or change what operators do, but....it is possible, which means that `== null` may not do what you think it should do. `is null` or `is not null` will do **exactly** what you think it should do.

Reply:
For example:
```csharp
var evil1 = new IAmEvil();
Console.WriteLine(evil1 == null);

public class IAmEvil
{
    public static bool operator ==(IAmEvil a, IAmEvil b) => true;
    public static bool operator !=(IAmEvil a, IAmEvil b) => true;
}
```

This will return `true`. Which is evil :).

Reply:
Oooo that makes sense, the example although evil is actually helpful thanks.